dungeonsheets/forms/empty_template.txt

```diff
@@ -21,7 +21,7 @@ background = "{{ char.background.name }}"
 race = "{{ char.race.name }}"
 alignment = "{{ char.alignment }}"
 
-xp = {{ char.xp }}
+xp = {{ char.xp }}  # ex: [36079] or ["Milestone"]
```

Comment:
Small change since it's just a comment, but I worry it will confuse people. The xp should be an int or a string, not a list.

Reply:
My bad, that comes from the upstream version, not your changes. Ignore this.